### PR TITLE
LPS-126224

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case is already covered by AssetPublisherUseCase#AssetPublisherFilterAndOrderByFieldType

**Comment by alexcandidos** [comment]
Hi @Tim-Cao , thanks for the review! Discussing with @emanuelfisilva , we'll keep them as is in the Data Engine scope.

**Comment by Tim-Cao** [comment]
Got it

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase [hunk 1]
```diff
@@ -125,12 +125,82 @@ definition {
...
+			category = "Content & Data",
+			portlet = "Web Content");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(
+			structureDescription = "Structure Description Test",
+			structureName = "Structure FieldSet Test");
+
+		DataEngine.addField(
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/translations/LocalizedFieldsetsWebContent.testcase:142]: Test fieldset not field

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3634
View total diff https://github.com/liferay-echo/liferay-portal/pull/3634/commits/a4deb5be1f5e3c58676ad18ed282fa9704c90059

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**